cli/cmd/component_dev.go

```diff
@@ -157,12 +157,9 @@ func runComponentsDevMode(_ *cobra.Command, args []string) error {
 			}
 
 		case "Python":
-			// TBA
-			cli.OutputHuman("\n (:sadpanda:) We are still working on a Python Scaffolding.")
-			cli.OutputHuman("\n\nIn the meantime use our Python SDK: %s\n",
-				color.HiMagentaString("https://github.com/lacework/python-sdk"))
-			cli.OutputHuman("And deploy your component at: %s\n",
-				color.HiYellowString(filepath.Join(rPath, component.Name)))
+			if err := cdkPythonScaffolding(component); err != nil {
```

Comment:
This makes me sooooooo happy!!! 

<img src="https://media0.giphy.com/media/l0amJzVHIAfl7jMDos/giphy.gif"/>